pipeline_dp/private_beam.py

```diff
@@ -104,3 +106,36 @@ def expand(self, pcol: pvalue.PCollection) -> pvalue.PCollection:
             value_extractor=self._sum_params.value_extractor)
 
         return dp_engine.aggregate(pcol, params, data_extractors)
+
+
+class Map(PrivatePTransform):
+    """Transform class for performing Map on PrivatePCollection."""
+
+    def __init__(self, fn: Callable, label: Optional[str] = None):
+        super().__init__(return_anonymized=False, label=label)
+        self._fn = fn
+
+    def expand(self, pcol: pvalue.PCollection):
+        return pcol | "map values" >> beam.Map(lambda x: (x[0], self._fn(x[1])))
+
+
+class FlatMap(PrivatePTransform):
+    """Transform class for performing FlatMap on PrivatePCollection."""
+
+    class _FlattenValues(beam.DoFn):
+        """Inner class for flattening values of key value pair.
+        Flattens (1, (2,3,4)) into ((1,2), (1,3), (1,4))"""
+
+        def process(self, row):
+            key = row[0]
+            values = row[1] if isinstance(row[1], Iterable) else [row[1]]
```

Comment:
It looks like [FlatMap](https://beam.apache.org/documentation/transforms/python/elementwise/flatmap/) accepts only iterable, let's also only support this case, no need to check here

Reply:
Done